### ISSUE TITLE
Possible false positive for CORE-000213 (CG0536)

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#568]
When working on a study for a customer, and using CORE (v.0.6.3) I got violations for CORE-000213 (CG0536) which I think are false positives, as the values for "EPOCH" are different.
Can someone have a look?
Screenshots attached.
![Unbenannt](https://github.com/cdisc-org/cdisc-rules-engine/assets/25853332/6e87556f-c405-4ca6-822c-49e003103e15)
![Unbenannt_2](https://github.com/cdisc-org/cdisc-rules-engine/assets/25853332/2a578fb7-a91c-49ba-84fc-28c9e740dcbf)

**Comment by nickdedonder** [comment]
With test data 02 this gives a positive result

**Comment by SFJohnson24** [comment]
@JozefAerts is this still an issue?  If not, we can close this issue.

**Comment by JozefAerts** [comment]
No, it is not an issue anymore. I tested again and all looks fine.
Ticket can be closed.
